Modules/TOF/src/TaskRaw.cxx

```diff
@@ -45,6 +45,11 @@ void RawDataDecoder::decode()
   DecoderBase::run();
 }
 
+void RawDataDecoder::rdhHandler(const o2::header::RAWDataHeader* rdh)
+{
+  mRDHCounter[rdh->feeId & 0xFF].Count(0);
```

Comment:
@preghenella I check the feeID like in the compressor. Then I fill the crate map with it in line https://github.com/AliceO2Group/QualityControl/pull/597/files#diff-85ba64596620595e54f7320c6e7707eaa1e851b0830a237a42f2aa56294e7696R434